scripts/openrouter-models/main.go

```diff
@@ -46,19 +65,37 @@ func main() {
 		panic(err)
 	}
 
-	var models []model.Model
+	models := map[string]bool{}
 	for _, model := range allModels {
 		if isIgnored(model) {
 			fmt.Printf("ignoring %q\n", model.ID())
 
 			continue
 		}
 
-		models = append(models, model)
+		// Skip base model if there is already an instruct tuned version.
+		if instruct := instructModel(models, model.ID()); instruct != "" {
+			fmt.Printf("ignoring %q for instruct model %q\n", model.ID(), instruct)
+
+			continue
+		}
+
+		if !isBaseModel(model.ID()) {
+			baseModelName := baseModel(model.ID())
+			fmt.Printf("(potentially) deleting %q for instruct model %q\n", baseModelName, model.ID())
+
+			delete(models, baseModelName)
+		} else if instructModelName := instructModel(models, model.ID()); instructModelName != "" {
+			fmt.Printf("ignoring %q for instruct model %q\n", model.ID(), instructModelName)
+
+			continue
+		}
```

Comment:
- The `if` below "Skip base model..." is not needed I think
- The "(potentially)" from the print statement can be removed if you do a map lookup first.
- maybe rename "instructModel" to "isInstructModelContained".